marketplace/marketplace.go

```diff
@@ -104,8 +105,9 @@ type Config struct {
 	MarketplaceNFTContract struct {
 		Address string `json:"address"`
 	} `json:"marketplaceNFTContract"`
-	RPCNodeAddress string `json:"rpcNodeAddress"`
-	ContractHash   string `json:"contractHash"`
+	RPCNodeAddress      string                `json:"rpcNodeAddress"`
+	ContractHash        string                `json:"contractHash"`
+	CasperTokenContract evmsignature.Contract `json:"casperTokenContract"`
```

Comment:
casper contract has type from evmsignature?

Reply:
redone to string